### ISSUE TITLE
Upgrade neighbourhood shapes for Athens, Greece

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#426]
### tl;dr

Status:
- [ ] Propose **sources**
- [ ] Establish **concordances** between WOF and source
- [ ] **Upgrade / add** WOF shapes
- [ ] **Add new or missing** features to WOF
- [ ] **Deprecate / supersede** WOF features
- [ ] **Draft review**
- [ ] **Draft revisions** based on review comments
- [ ] **Final review**
- [ ] **Final revisions** based on review comments
- [ ] **Import** into Who's On First

### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by stepps00** [comment]
per @okavvada:

[This site](http://geodata.gov.gr/en) has most of the open data for Greece. No neighbourhood-specific data, but [this dataset](http://geodata.gov.gr/en/group/6bae1869-da23-4edb-b6b4-555131be28f0?organization=orsa) contains all of Athens' official planning boundaries (Cadastre), which are essentially outlines of city block (very similar to neighbourhoods).

Data is open through [this license](https://creativecommons.org/licenses/by/3.0/).

**Comment by nvkelso** [comment]
Neat! But when I download one of those files, it has tons of stuff but all
in Greek. @okavvada, can you point to specific files to use, please?

**Comment by okavvada** [comment]
* Municipality boundaries: http://geodata.gov.gr/en/dataset/oria-demon-kallikrates

Should be straight forward, it includes the boundaries and the municipality name in Greek :( (I had to download the csv file and join that back to the shapefile in order to read the Greek characters)
* Urban plans (mostly for Athens): http://geodata.gov.gr/en/dataset?tags=%CF%80%CE%BF%CE%BB%CE%B5%CE%BF%CE%B4%CE%BF%CE%BC%CE%B9%CE%BA%CE%AC+%CF%83%CF%87%CE%AD%CE%B4%CE%B9%CE%B1&groups=boundaries

All the filenames are in Greek (sorry :( ). The filename includes a description of the file and the last word is the municipality name that it belongs to. A translation of the different filenames is here:

athlitismos_koridallos = **sports** of the municipality of Koridallos
ekpaidefsi_koridallos = **education** of the municipality of Koridallos
katoikia_koridallos (in other files this is also listed as `amigis_katoikia`, `mikti_katoikia`, `gen_katoikia`, `geniki_katoikia`, `amigis_kat` - in general anything with the word `katoikia`) = **housing** of the municipality of Koridallos
kentro_dimou_koridallos = **center** of the municipality of Koridallos
kentro_geit_koridallos (or `kentro_geitonias`) = **center of neighborhood** of the municipality of Koridallos
odiko_koridallos = **road network** of the municipality of Koridallos
oria_geitonias_koridallos = **neighborhood boundaries** of the municipality of Koridallos
oria_synoikias_koridallos = **district boundaries** of the municipality of Koridallos
orio_egek_koridallos (or `orio_egekrimenou) = **approved boundary** of the municipality of Koridallos
orio_epektasis_koridallos = **extension boundary** of the municipality of Koridallos
orio_gps_koridallos = **gps boundary** of the municipality of Koridallos
politistika_koridallos = **cultural** of the municipality of Koridallos
prasino_koridallos = **green areas** of the municipality of Koridallos
pronoia_koridallos = **care facilities** of the municipality of Koridallos

**Comment by okavvada** [comment]
There are also other cool stuff in there like:
* dams - http ://geodata.gov.gr/en/dataset/fragmata-elladas
* transit - http://geodata.gov.gr/en/dataset/oasa
* airports - http://geodata.gov.gr/en/dataset/aepodpou1a-ejjadas
* protected areas - http://geodata.gov.gr/en/dataset/to-diktuo-natura-2000-kai-prostateuomenes-periokhes
* railway - http://geodata.gov.gr/en/dataset/siderodromiko-diktuo-elladas
* points of interest -http://geodata.gov.gr/en/dataset/semeia-endiapherontos-tou-demou-kalamarias

and more...

**Comment by nvkelso** [comment]
Thanks for the walk thru! Now I want to visit :)
